src/main/java/games/strategy/engine/chat/HeadlessChat.java

```diff
@@ -165,7 +163,7 @@ public void addMessageWithSound(final String message, final String from, final b
 
   private void addChatMessage(final String originalMessage, final String from, final boolean thirdperson) {
     final String message = trimMessage(originalMessage);
-    final String time = simpleDateFormat.format(new Date());
+    final String time = "(" + TimeManager.getLocalizedTime() + ")";
```

Comment:
:+1: Appears to be the same analysis as for `ChatMessagePanel#addChatMessage()`.